app/views/govuk_publishing_components/components/_image_card.html.erb

```diff
@@ -27,6 +27,7 @@
 <% if card_helper.href || card_helper.extra_details.any? %>
   <div class="<%= classes %> <%= brand_helper.brand_class %>"
     <%= "data-module=gem-track-click" if card_helper.is_tracking? %>
+    <%= "data-module=image-card" if card_helper.youtube_video_id %>
```

Comment:
If tracking is enabled (on the previous line) then `data-module` is being declared twice for a youtube video. Will this work or will browsers ignore the second one? This might need to combine these two declarations.

Reply:
Yep, this will overwrite 'gem-track-click' so I'll move it to another element.

Reply:
actually https://github.com/alphagov/govuk_publishing_components/pull/3154 if this gets merge i guess i can use the same element

Reply:
if gets merged in https://github.com/alphagov/govuk_publishing_components/pull/3154/files would use this implementation